coordinator/src/notification_service.rs

```diff
@@ -130,3 +144,205 @@ async fn send_notification<'a>(
     tracing::debug!("Sent: {:?}", response);
     Ok(())
 }
+
+/// Send notifications to users with positions that are about to expire or have
+/// just expired
+pub async fn send_expiry_notifications_if_applicable(
+    positions: &[(Position, FcmToken)],
+    notification_sender: &mpsc::Sender<Notification>,
+) {
+    let now = OffsetDateTime::now_utc();
+
+    for (position, fcm_token) in positions {
+        if position.expiry_timestamp <= now
+            && now < position.expiry_timestamp + time::Duration::hours(1)
+        {
+            if let Err(e) = notification_sender
```

Comment:
🔧 With #1246 we are creating an `AsyncOrder` for expired positions and try to send them when processing the new order. Currently we only try through the websocket, but if the user is not online we should send the push notification. 

Please add this push notification there as well, once #1246 is merged.

https://github.com/get10101/10101/pull/1246/files#diff-069f682364e37633796656003287aac144d258002b4f2f2d7933b6a94dcc5010R249

Reply:
this will be covered by #1211

Reply:
Added it to the same PR, as it turned out to be a small task.